File: CodeForces/732A/732A.go
package main

import (
    "fmt"
)

func main() {
    var cost,coin int
    fmt.Scan(&cost,&coin)

	var attempt int = 1

    for {
		if (cost*attempt) % 10 == 0 || ((cost*attempt)-coin) % 10 == 0{
			fmt.Println(attempt)
			return
		}
		attempt = attempt + 1
	}
}

File: CodeForces/701-800/732/A/732A.py
# Problem 732A: Buy a Shovel
# https://codeforces.com/contest/732/problem/A
# Executed with: Python 3.11.2
# Linux version: Linux kernel 6.1.0-10-amd64
# Submitted on: September 27th, 2024
# Codeforces language used: PyPy 3.10 (7.3.15, 64bit)

cost, coin = map(int, input().split())

testing_amount = 1

while True:
    if cost*testing_amount % 10 == 0 or ((cost*testing_amount)-coin) % 10 == 0:
        print(testing_amount)
        break
    testing_amount += 1

